configs/sst_nic_rdma-unwanted.yaml

```diff
@@ -0,0 +1,42 @@
+document: feedback-pipeline-workload
```

Comment:
This needs to be `document: feedback-pipeline-unwanted` for an exclusion list

Reply:
Fixed. thanks